### DISCUSSION TITLE
Gluon mobile

### DISCUSSION THREAD

**Discussion by ZHAIFEIXIANG** [gluonhq/scenebuilder#793] [General]
Hi Gluon team
We are from Income Insurance https://www.income.com.sg/, which is one of the leading composite insurers in Singapore. We are currently exploring alternate solution to enhance our current mobile framework- Oracle MAF, and we are interested in evaluating Gluon mobile as a potential replacement for our existing mobile solution. We would appreciate the opportunity to trial Gluon solution. This would enable us to assess the expected capabilities and determine its fit for our criteria needs. Hope you could assist us in setting up a trial account for us to proceed on our R&D evaluation.

Thanks for considering our request. I look forward to your positive response.

**Comment by Oliver-Loeffler** [comment]
Hi  @ZHAIFEIXIANG, I think best way to contact Gluon is via their contact page. 
https://gluonhq.com/about-us/contact-us/

@johanvos @jperedadnr
